src/build.sh

```diff
@@ -209,7 +233,9 @@ function build_help()
     '  build - Build kernel' \
     '  build (-n | --menu) - Open kernel menu config' \
     '  build (-i | --info) - Display build information' \
-    '  build (-d | --doc) - Build kernel documentation'
+    '  build (-d | --doc) - Build kernel documentation' \
+    '  build (-c | --cpu-scaling) <percentage> - Scale CPU usage by factor' \
+    '  build (--ccache) - Enable use of ccache'
```

Comment:
Just for curiosity, what will happen in these scenarios:
1. `kw b -n -c`
2. `kw b -n --ccache`
3. `kw b -d --ccache`
4. `kw b -i -c`
5. `kw b -i --ccache`

Reply:
> 1. `kw b -n -c`
> 2. `kw b -n --ccache`

`-n` will not take any of these flags into consideration.

> 3. `kw b -d --ccache`

This should take `ccache` into consideration just fine :)

> 4. `kw b -i -c`
> 5. `kw b -i --ccache`

I'm really not sure about the `-i` flag, as I've not messed with it at all.